src/pages/resources/howtos/running-network-infrastructure.md

```
@@ -0,0 +1,175 @@
+
+!!! info Tuning the bootstrap server's performance
+You can pass various parameters to `kitsune2-bootstrap-srv` to tune the relay performance. The [`sbd_server` crate documentation](https://docs.rs/sbd-server/latest/sbd_server/struct.Config.html#structfield.limit_clients) shows the data structure; the tuning parameters are `limit_clients` onward. To pass them as arguments to the Docker container, prefix them with `sbd` and convert them to hyphen-case, like this:
+
+```yaml
+# ...
+    command:
+      - kitsune2-bootstrap-srv
+      - --sbd-limit-clients
+      - 50
+# ...
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)